packages/web/src/components/track-availability-modal/TrackAvailabilityModal.tsx

```diff
@@ -144,18 +151,35 @@ const TrackAvailabilityModal = ({
   const numEthCollectibles = Object.keys(ethCollectionMap).length
   const numSolCollectibles = Object.keys(solCollectionMap).length
   const hasCollectibles = numEthCollectibles + numSolCollectibles > 0
-  const noCollectibleGate = !hasCollectibles || isRemix
-  const noCollectibleDropdown = !hasCollectibles || isRemix || !isUpload
-  const noSpecialAccess = isRemix || !isUpload
+
+  const initialPremiumConditions = initialForm.premium_conditions
+
+  const isInitiallySpecialAccess =
+    !isUpload &&
+    !!(
+      initialPremiumConditions?.follow_user_id ||
+      initialPremiumConditions?.tip_user_id
+    )
+  const noCollectibleGate =
+    isInitiallySpecialAccess || isRemix || !hasCollectibles
+  const noCollectibleDropdown = noCollectibleGate || !isUpload
+
+  const isInitiallyCollectibleGated =
```

Comment:
nit: feels a bit like a misnomer, maybe just combine this into one boolean expression in the next line